Gemfile

```
@@ -7,29 +7,21 @@ branch = ENV.fetch('SOLIDUS_BRANCH', 'main')
 gem 'solidus', github: 'solidusio/solidus', branch: branch
 
 # The solidus_frontend gem has been pulled out since v3.2
-if branch >= 'v3.2'
-  gem 'solidus_frontend'
-elsif branch == 'main'
-  gem 'solidus_frontend', github: 'solidusio/solidus_frontend'
-else
-  gem 'solidus_frontend', github: 'solidusio/solidus', branch: branch
-end
+gem 'solidus_frontend', github: 'solidusio/solidus_frontend'
 
-# Needed to help Bundler figure out how to resolve dependencies,
-# otherwise it takes forever to resolve them.
-# See https://github.com/bundler/bundler/issues/6677
-gem 'rails', '>0.a'
-
-# Provides basic authentication functionality for testing parts of your engine
-gem 'solidus_auth_devise'
+rails_requirement_string = ENV.fetch('RAILS_VERSION', '~> 7.0')
+gem 'rails', rails_requirement_string
 
 case ENV['DB']
 when 'mysql'
   gem 'mysql2'
 when 'postgresql'
   gem 'pg'
 else
-  gem 'sqlite3'
+  rails_version = Gem::Requirement.new(rails_requirement_string).requirements[0][1]
+  sqlite_version = rails_version < Gem::Version.new(7.2) ? "~> 1.4" : "~> 2.0"
```

Can we simplify this to?

```suggestion
  rails_version = Gem::Version.new(rails_requirement_string)
  sqlite_version = rails_version < Gem::Version.new(7.2) ? "~> 1.4" : "~> 2.0"
```

---

No, because the circleci image specifies the rails version as `~> 7.0`...